src/backend/mailbox_manager/admin.py

```diff
@@ -92,6 +92,7 @@ class MailDomainAdmin(admin.ModelAdmin):
 
```

Comment:
Je te propose de supprimer la property et de juste rajouter dans la `class MailDomainAdmin` la méthode suivante qui fait le job après avoir renommé le related_name ;)
```suggestion
def count_mailboxes(self, obj):
    return obj.mailboxes.count()
```